src/amo/sagas/index.js

```diff
@@ -12,10 +12,10 @@ import addons from 'core/sagas/addons';
 
 // Export all sagas for this app so runSaga can consume them.
 export default function* rootSaga() {
-  yield [
+  yield all([
```

Comment:
Was this necessary? I think `fork()` already instructs the middleware to make a non-blocking call.